ltp_config/manifest_18_04.template

```diff
@@ -41,6 +41,8 @@ sgx.allowed_files = [
     "file:install-sgx/testcases/bin/execv01_child",
     "file:install-sgx/testcases/bin/execlp01_child",
     "file:install-sgx/testcases/bin/execl01_child",
+    "file:install-sgx/testcases/bin/execle01_child",
```

Comment:
If execle01 and execve01 is not enabled, then why these files has been added to allowed files

Reply:
Removed